services/app-api/prisma/migrations/20250627111243_add_dsnp_field_to_contract_revs/migration.sql

```diff
@@ -0,0 +1,17 @@
+BEGIN;
+/*
```

Comment:
@mojotalantikite I wanted to flag you on this migration due to the `DocumentZipType` changes. It popped up when I ran prisma for my migration to alter the `ContractRevisionTable`

https://github.com/Enterprise-CMCS/managed-care-review/pull/3354/files#diff-c8bae96602bf4ad389bfce681646017d7368430a95cf18bf088515adab3f8dbc